Prox/Prox/PlaceCarousel/PlaceCarouselViewController.swift

```diff
@@ -193,7 +198,12 @@ class PlaceCarouselViewController: UIViewController {
         // apply the constraints
         NSLayoutConstraint.activate(constraints, translatesAutoresizingMaskIntoConstraints: false)
 
-        toggleLoadingUI(loading: true)
+        toggleLoadingUI(loading: shouldFetchPlaces)
```

Comment:
Just a heads up that this will conflict with my loading screen->detail animation patch as I've replaced this method with a `isLoading` property https://github.com/mozilla-mobile/prox/pull/373